src/components/sdk/DeviceSelection/docs/VideoFilterSelection.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import VideoFilterSelection from '../CameraSelection/VideoFilterSelection';
+
+<Meta title="SDK Components/DeviceSelection/Camera/VideoFilterSelection" />
+
+# VideoFilterSelection
+
+The `VideoFilterSelection` component renders a `Checkbox FormField` to select a background blur device. Currently, the only option is background blur.
```

Comment:
Do we want to callout how this works with `PreviewVideo` as well as `LocalVideo` ? Also, suggest adding `PreviewVideo` component in the usage example. It might help builder in knowing when to use this component.

Reply:
This component, I think is very close to `VideoInputBackgroundControl` as well so we should provide some guidance on the differences. Also, one more question, since actually both `VideoInputBackgroundControl` and now the `VideoFilterSelection` are just selecting the transform devices just that the UI is different, should we or can we implement or re-use the creation and changing of video transform device or normal video input device and not repeat in two places?

Reply:
hmm I think that would be possible, maybe I can put this component in the VideoInputBackgroundControl later.